### ISSUE TITLE
Add skydive

### ISSUE THREAD

**Issue by sivakom** [openvswitch/ovn-scale-test#71]
Install the skydive controller into it’s own container, and also add the skydive agents into each of the containers running ovn-controller.

**Comment by huikang** [comment]
@sivakom could you elaborate more in this proposal? Thanks. - Hui

**Comment by hzhou8** [comment]
@sivakom thanks for the proposal, but I have the same question as Hui. Could you clarify the reason why we need skydive in scale-test env? Would skydive help provide data for scalability?

**Comment by sivakom** [comment]
Yes definitely. Skydive provides an abundance of data regarding the topology flows and information from all of the ovn-controller chassis, which is then aggregated into topology graphs in the skydive analyzer. This allows us to see how all of the various network components are interacting amongst each other, especially in such a large scale. Feel free to add anything that I might have missed @mestery.

**Comment by huikang** [comment]
@sivakom thanks for the explanation. I am looking forward to the implementation.

**Comment by hzhou8** [comment]
@sivakom so the purpose is to see how skydive would work in a large scale deployment. I think it is valuable 👍
